packages/kmod/kmod.spec

```diff
@@ -27,6 +27,27 @@ cp COPYING COPYING.LGPL
 cp tools/COPYING COPYING.GPL
 
 %build
+
+mkdir static-build
+pushd static-build
+
+%define _configure ../configure
+export LDFLAGS="-static -lz -lzstd"
```

Comment:
Do we need this? Since we're overriding LDFLAGS during the make.

The actual issue here is that we end up not using our standard hardened linker flags, which we could fix by just not setting this.

Reply:
Nope. Tested that we don't need these. Removing them. Thanks!